packages/web/src/components/gcds-fieldset/gcds-fieldset.tsx

```diff
@@ -251,7 +253,11 @@ export class GcdsFieldset {
     this.validateValidator();
 
     // Assign required validator if needed
-    requiredValidator(this.el, 'fieldset');
+    if (this.el.getAttribute('data-date-input')) {
```

Comment:
Could we name the attribute just `date-input` or even just `date`? As far as I know, `date` is not an existing HTML attribute.

Reply:
I wonder if `data-date` is better? I know `data-*` is for [custom data attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*) so hopefully we can keep that part. Because of that I think we can also drop `*-input`.
Other options:
- `data-value`
- `data-date-value`

Reply:
Oops, I read `date-date`, `data-date` makes a lot more sense haha